bokehjs/src/coffee/models/annotations/legend.coffee

```diff
@@ -91,7 +91,7 @@ export class LegendView extends AnnotationView
   render: () ->
     if not @model.visible
       return
-  
+
```

Comment:
I highly recommend enabling trailing whitespace trimming in your code editor. You can also run `py.test -m quality` locally before pushing changes. This will spare you some work and leave travis ci's resources for more interesting changes.